tests/sync_object.rs

```diff
@@ -1,6 +1,15 @@
 #[allow(dead_code)]
 fn needs_sync<T: Sync>() {}
 
+#[test]
+fn event_loop_proxy_send() {
+    #[allow(dead_code)]
+    fn is_send<T: 'static + Send>() {
+        // ensures that `winit::EventLoopProxy` implements `Send`
+        needs_sync::<winit::event_loop::EventLoopProxy<T>>();
+    }
```

Comment:
This should just use `needs_sync`, because we test `Send` in other test.

Reply:
This doesn't test if `EventLoopProxy` implements `Send`, it tests if `EventLoopProxy` implement `Sync` if `T` is `Send`.

Reply:
But you can move the `T: Send` in the test itself?

Reply:
I don't think you can use generics on `#[test]`. It would be a compile error anyway so we don't *really* needs tests in this sense.

I think the way it's right now is just fine, but maybe adding a comment would help.

Reply:
Not sure what I should write as a comment, but I've made the exiting comment text a bit clearer